bin/kano-dev

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+
+# kano-dev
+#
+# Copyright (C) 2016 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+#
+# Run common dev commands. Expects to run as root
+
+"""
+kano-dev runs common dev commands on the pi
+
+Usage:
+  kano-dev rootssh
+  kano-dev production (all|api|apt)
+  kano-dev staging (all|api|apt)
+  kano-dev scratch
+
+ Options:
+"""
+
+import docopt
+import os
+
+KANO_CONTENT_STAGING = "api-url: http://kano-content-api-test.herokuapp.com\n"
+
+KANO_WORLD_STAGING = """api_url: http://kano-api-staging.herokuapp.com
```

Comment:
Thanks @convolu ! So we switched back again. What about the content api, is that back as well?